dist/index.html

```diff
@@ -1,24 +1,24 @@
-<!DOCTYPE html>
-<html>
-  <head>
-    <meta charset="utf-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <meta name="viewport" content="width=device-width,initial-scale=1" />
-    <title>Output Management</title>
-    <script defer="defer" src="runtime.bundle.js"></script>
-    <script defer="defer" src="index.bundle.js"></script>
-  </head>
-  <body>
-    <div class="container">
-      <div class="list">
-        <h2>To-do list</h2>
-        <div class="row">
-          <input id="input-box" placeholder="add to your list" />
-          <button onclick="addToList()">Add</button>
-        </div>
-        <ul id="list-container"></ul>
-        <button onclick="removeList()" class="removeButton">clear list</button>
-      </div>
-    </div>
-  </body>
-</html>
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width,initial-scale=1" />
+    <title>Output Management</title>
+    <script defer="defer" src="runtime.bundle.js"></script>
+    <script defer="defer" src="index.bundle.js"></script>
+  </head>
+  <body>
+    <div class="container">
+      <div class="list">
+        <h2>To-do list</h2>
+        <div class="row">
+          <input id="input-box" placeholder="add to your list" />
+          <button onclick="addToList()">Add</button>
+        </div>
+        <ul id="list-container"></ul>
+        <button onclick="removeList()" class="removeButton">clear list</button>

```

Comment:
- Clicking on the clear all button, it suppose to only clear tasks that are marked as completed, however when I clicked on it cleared all tasks, which goes contrary to the project requirement. Please make the triggered function clear only tasks that are marked as completed.